client/my-sites/comments/comment-list/index.jsx

```diff
@@ -143,8 +149,14 @@ export class CommentList extends Component {
 
 	updateLastUndo = commentId => this.setState( { lastUndo: commentId } );
 
+	getCommentsCount = ( commentCounts, status ) => {
```

Comment:
What if we moved this to mapStateToProps, and expose a single count in props for now?